### ISSUE TITLE
Collaborator to become contributor

### ISSUE THREAD

**Issue by magdalenadrafiova** [DMPRoadmap/roadmap#2823]
Please complete the following fields as applicable:

**What version of the DMPRoadmap code are you running? v.2.2.0

It has been raised that currently there is a new tab with contributors. 
However when the user shares a plan with the collaborators for editing, read-only or co-owner they are not automatically added to the contributors tab. 
It would worth considering triggering automatic contributors action, if the plan is shared for co-owners or editing purposes.

**Comment by briri** [comment]
see #2481

**Comment by mariapraetzellis** [comment]
Agreed, it would be nice to have a feature that gives users the option of automatically adding contributors to the collaborators list. This should be an opt in feature, since not all contributors to a plan need to be involved in creating the DMP.
